oauth2client/contrib/gce.py

```diff
@@ -58,6 +58,9 @@ def __init__(self, scope, **kwargs):
             scope: string or iterable of strings, scope(s) of the credentials
                    being requested.
         """
+        # if scope:
+        #     raise ValueError('GCE will not respect your scopes so what '
+        #                      'is the point?')
         self.scope = util.scopes_to_string(scope)
```

Comment:
Is this a stray commit?  If we truly cannot usefully pass in the `scope`, then why not remove it altogether?